System/Metrics/Prometheus/Example.hs

```diff
@@ -17,11 +17,11 @@ import qualified System.Metrics.Prometheus.Gauge as Gauge
 import System.Metrics.Prometheus
 
 -- Custom type describing a set of classes of metrics.
-data MyMetrics (name :: Symbol) (t :: MetricType) (labels :: Type) where
```

Comment:
It was nice having the key names, but not a big deal.

Reply:
I've now added the key names back as comments